llvm/test/MC/RISCV/rv32p-valid.s

```diff
@@ -1,5 +1,5 @@
 # RUN: llvm-mc %s -triple=riscv32 -mattr=+experimental-p -M no-aliases -show-encoding \
-# RUN:     | FileCheck -check-prefixes=CHECK-ASM,CHECK-ASM-AND-OBJ %s
+# RUN:     | FileCheck -check-prefixes=CHECK-ASM %s
```

Comment:
Please add back CHECK-ASM-AND-OBJ. Add a additional CHECK-OBJ to the other RUN line that uses CHECK-ASM-AND-OBJ. Then use CHECK-ASM and CHECK-OBJ to show the different output for the case that fails.